videostore/concat.c

```diff
@@ -74,8 +75,6 @@ int video_store_concat(const char *concat_filepath, const char *output_path) {
              i, av_err2str(ret));
       goto cleanup;
     }
-    // Let ffmpeg handle the codec tag for us.
-    outStream->codecpar->codec_tag = 0;
```

Comment:
we need to remove this or the `codec_tag` written by rawsegmenter.c (which is needed for quicktime to be able to play the produced video) will be removed from the concated video.